vcpkg.json

```diff
@@ -48,17 +53,18 @@
         "libmysql"
       ]
     },
+    "simdutf": {
+      "description": "Enable SIMD UTF-8 support",
+      "dependencies": [
+        "simdutf"
+      ]
+    },
     "unit-tests": {
       "description": "Build unit tests",
       "dependencies": [
         "boost-test"
       ]
     }
   },
-  "default-features": [
-    "lua",
-    "http",
-    "libmariadb"
-  ],
```

Comment:
I have to remove this because it is impossible to _disable_ a feature from CMake, so we can't choose one or the other, only add to the default feature set.